config/database.yml

```diff
@@ -1,28 +1,16 @@
-# SQLite version 3.x
-#   gem install sqlite3
-#
-#   Ensure the SQLite 3 gem is defined in your Gemfile
-#   gem 'sqlite3'
-#
 default: &default
```

Comment:
You shouldn't upload the database.yml, because Docker needs a different configuration.